### ISSUE TITLE
zsys being removed from ubuntu

### ISSUE THREAD

**Issue by devZer0** [ubuntu/zsys#235]
ubuntu considered dropping support for zsys

https://bugs.launchpad.net/ubuntu/+source/ubiquity/+bug/1968150

this sucks

please help making sure zsys is being maintained and be part of ubuntu. it's useful

**Comment by lciti** [comment]
Would [zfsbootmenu](https://zfsbootmenu.org/) be a good replacement? I have tried it and sort of works (fine in my desktop; so and so in my laptop due to issues with some drivers). I have used the pre-compiled boot kernels (see EFI files [here](https://github.com/zbm-dev/zfsbootmenu/releases)) which is more straightforward than trying to get ubuntu build the boot initramfs. A standard zfs snapshotting tool with retention (e.g. [zfs-autobackup](https://github.com/psy0rz/zfs_autobackup) or [zfs-auto-snapshot](https://github.com/zfsonlinux/zfs-auto-snapshot)) would do the work of creating the snapshots, while zfsbootmenu would present those snapshots as boot options in a menu at boot time.
